packages/wayland/compositor/sway/patches/sway-100.01-static-ipc-socket.patch

```diff
@@ -0,0 +1,12 @@
+--- /sway/ipc-server.c
```

Comment:
Should this have a git header?

Reply:
`./tools/refresh-patches sway`
to fix this i guess

Reply:
it's a hacky patch by me to avoid adding the pid to the socket path - I didn't use git or else to create it.

Reply:
`./tools/refresh-patches sway`

Reply:
done